api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/core/Permute.kt

```diff
@@ -0,0 +1,55 @@
+package org.jetbrains.kotlinx.dl.api.core.layer.core
```

Comment:
To be more precise, the Permute layer is in the category of "Reshaping" layers; so it could be moved to that directory instead.